CHANGELOG.md

```diff
@@ -8,8 +8,14 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ### Changed
 
+- The `wasmParallel` `Feature` has been the replaced by the corresponding `WebFeature`s. A caller only needs to pass the set of available features to the `getAssets` function and the library automatically returns the most suitable WASM assets for the given features.
 - MAB system is not executed anymore. Only the context value is used instead to rank the documents.
 
+### Removed
+
+- Removed the optional `features` parameter in the `SetupData` constructor.
```

Comment:
This is a breaking change, correct?

Reply:
(Yes?, Through did we ever ship a version with that parameter included?)